aws-distributed-configuration/src/main/java/io/micronaut/aws/distributedconfiguration/AwsDistributedConfiguration.java

```diff
@@ -34,6 +35,13 @@ public interface AwsDistributedConfiguration {
     @NonNull
     String getPrefix();
```

Comment:
Please, also mention in `getPrefix()` Javadoc that it will not be used if prefixes is set.